collect-logs

```diff
@@ -89,6 +93,12 @@ class Juju(object):
         self.env = None
         if cfgdir is not None:
             self.env = dict(os.environ, **{self.envvar: cfgdir})
+            self.ssh_key = os.path.join(cfgdir, "ssh", "juju_id_rsa")
+        else:
+            default_cfgdir = os.path.join(
+                os.path.expanduser("~"), ".local", "share", "juju")
```

Comment:
Please add a comment that this is assuming juju2 and juju1 will not work with this. I don't think we need to make juju 1 work, just note it.